### PR TITLE
[Survey]: pep8 naming conventions for series_export_spreadsheet

### PR DESCRIPTION
**Pull request by raj454raj**

### CODE REVIEW

File: controllers/survey.py [hunk 1]
```diff
@@ -567,62 +557,62 @@ def wrapText(sheet, cell, style):
             sheet.write(cell.row,
                         cell.col,
                         text,
-                        style
+                        style,
                         )
             rows = math.ceil(len(text) / characters_in_cell)
         new_row_height = int(rows * twips_per_row)
-        new_col_width = width * COL_WIDTH_MULTIPLIER
+        new_col_width = width * col_width_multiplier
```
Comment (nursix) [inline controllers/survey.py]: Constants like this should use all-uppercase, i.e. COL_WIDTH_MULTIPLIER is actually correct.

File: controllers/survey.py [hunk 2]
```diff
@@ -567,62 +557,62 @@ def wrapText(sheet, cell, style):
...
             except:
-                if zeroObj.__dict__[element] != value:
-                    baseObj.__dict__[element] = value
-        return baseObj
+                if zero_obj.__dict__[element] != value:
+                    base_obj.__dict__[element] = value
+        return base_obj
 
-    COL_WIDTH_MULTIPLIER = 240
+    col_width_multiplier = 240
```
Comment (nursix) [inline controllers/survey.py]: Should be COL_WIDTH_MULTIPLIER, see above

File: controllers/survey.py [hunk 3]
```diff
@@ -630,149 +620,149 @@ def mergeObjectDiff(baseObj, newObj, zeroObj):
...
+    style_list["boxR1"] = box_r1
+    style_list["boxR2"] = box_r2
+    style_list["boxB1"] = box_b1
+    style_list["boxB2"] = box_b2
 
     sheet1 = book.add_sheet(T("Assessment"))
-    sheetA = book.add_sheet(T("Metadata"))
-    maxCol = 0
+    sheet2 = book.add_sheet(T("Metadata"))
+    max_col = 0
```
Comment (nursix) [inline controllers/survey.py]: Same here: should be MAX_COL (all-uppercase) to indicate it's a constant (well, there is not really such a thing like a constant in Python, but we see hardcoded values like this as equivalent).

File: controllers/survey.py [hunk 4]
```diff
@@ -785,40 +775,40 @@ def mergeObjectDiff(baseObj, newObj, zeroObj):
...
-        sheet1.col(col).width = cellWidth
-
-    sheetA.write(0, 0, "Question Code")
-    sheetA.write(0, 1, "Response Count")
-    sheetA.write(0, 2, "Values")
-    sheetA.write(0, 3, "Cell Address")
-    for cell in matrixAnswers.matrix.values():
-        style = mergeStyles(styleList, cell.styleList)
-        sheetA.write(cell.row,
+    cell_width = 480 # approximately 2 characters
```
Comment (nursix) [inline controllers/survey.py]: Same here: CELL_WIDTH

File: controllers/survey.py (hunk 1 above)
Comment (raj454raj) [inline controllers/survey.py]: :+1:

**Comment by nursix** [comment]
Looks largely ok - just the minor issue with constant names.

I hope you have tested it?

**Comment by raj454raj** [comment]
Yeah :+1:

File: controllers/survey.py (hunk 3 above)
Comment (raj454raj) [inline controllers/survey.py]: I don't max_col should be upper case. This variable which can be mapped to something like a max variable to find maximum of a list and keep updating while iterating through the list. What say? @nursix 
https://github.com/nursix/eden/pull/34/files#diff-5b17b6ab7d250b22d0d215163c979b30R739
https://github.com/nursix/eden/pull/34/files#diff-5b17b6ab7d250b22d0d215163c979b30R783

**Comment by raj454raj** [comment]
@nursix : Done. Kept max_col as it is?

**Pull request merged**